src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,146 @@
+import axios from 'axios'
+import { lang } from 'botpress/shared'
+import _ from 'lodash'
+import moment from 'moment'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+import { EventPackageInfoType, StoreInfoType, TelemetryPackage } from './telemetry_type'
+
+const telemetryPackageVersion = '1.0.0'
```

Comment:
I think it would be safer to send this from the backend along with telemetry data as we don't want to change those things at 2 places